map.js

```diff
@@ -101,9 +123,95 @@ function zoomMap(event){
     if(event.deltaY > 0) zoomLevel += 0.2;
     else zoomLevel += -0.2;
     
+    //TODO: make it so that it zooms into middle of screen rather than top left corner?
+
     //clamp zoom
     if(zoomLevel > maxZoom) zoomLevel = maxZoom;
     if(zoomLevel < minZoom) zoomLevel = minZoom;
-    console.log(zoomLevel);
     moveMap();
+}
+
+//attaches width to width of iframe
+function iFrameCheck(){
```

Comment:
why is this function? plz explain.

Reply:
iFrame would initialize after the canvas. so this was my work around for a "delay" that would keep checking until the iframe is done.